test/models/query.spec.js

```diff
@@ -0,0 +1,130 @@
+import chai from 'chai';
+
+import Query from '../../src/models/query';
+import QueryAggregate from '../../src/models/queryAggregate';
+
+describe('Query', function() {
+  describe('Construct', function() {
+    it('should create a basic object with empty fields, aggregates, filters and sort', function() {
+      const queryName = 'test_query';
+      const query = new Query(queryName);
+      chai.expect(query.name).to.equal(queryName); // eslint-disable-line
```

Comment:
remove this eslint disbale and disable this rule in the config file because it doesn't fit with chai.